.github/workflows/build.yml

```diff
@@ -105,15 +105,14 @@ jobs:
           name: python-wheel-license
           path: LICENSE.txt
 
-  build-python-mac-win:
+  build-windows:
```

Comment:
I'm not sure why you removed mac from here? is there other job which tests this python on mac